src/bp/ui-admin/src/App/UserDropdownMenu.jsx

```diff
@@ -19,6 +19,14 @@ class UserDropdownMenu extends Component {
     !this.props.profile && this.props.fetchProfile()
   }
 
+  gotoServer = () => {
```

Comment:
I think this logic should not reside here as it might (probably will have to be used somewhere else)

It could be handled by the Server page itself with a route listener. Everytime someone's trying to render the Server page, the listener will simply push the user to the appropriate tab given the licensing pro status

Reply:
It would be a good idea, but i'm not a fan of connecting the TabLayout module to the store. It's actually a pure component that only displays the sub page, do we really want to make it a stateful component?